src/proc/mod.rs

```diff
@@ -1,4 +1,4 @@
 mod interface;
-mod typifier;
+pub(crate) mod typifier;
```

Comment:
let's be consistent here and just add the types you need in the following `pub use` statement instead of switching the visibility of this module

Reply:
👍